doc/herbstluftwm.txt

```diff
@@ -850,6 +850,9 @@ set_attr 'ATTRIBUTE' 'NEWVALUE'::
     Assign 'NEWVALUE' to the specified 'ATTRIBUTE' as described in the
     <<OBJECTS,*OBJECTS section*>>.
 
+type_of_attr 'ATTRIBUTE'::
+    Print (one line containing) the type of the specified 'ATTRIBUTE'.
```

Comment:
Is the (one line containing) necessary? It kind of reads "bumpy", if you know what I mean

Reply:
Hm, I think I can also remove it. It will be important that the output is precisely one line :)

Reply:
Thanks! Reads better now IMHO. If it is important we can add it, maybe the only thing making it "bumpy" were the parantheses

Reply:
‘a’ instead of ‘one’ would also make it read better (IMO)

Reply:
I've removed the addition in the parentheses already.

Reply:
Closing this now :)